src/Client.ts

```diff
@@ -577,6 +620,27 @@ export default class Client {
     )
   }
 
+  /**
+   * List raw message bytes of envelopes stored on a specific topic
+   *
+   * Useful for Voodoo testing
+   */
+  async listRawVoodoo(topic: string): Promise<Uint8Array[]> {
```

Comment:
This also looks a whole lot like `listEnvelopes`, just without the mapping function (which could be as simple as `(env) => env`)